src/support_code_library/builder.js

```diff
@@ -13,23 +13,33 @@ function build({cwd, fns}) {
     defaultTimeout: 5000,
     listeners: [],
     stepDefinitions: [],
-    transformLookup: TransformLookupBuilder.build(),
+    parameterRegistry: TransformLookupBuilder.build(),
     World({attach, parameters}) {
       this.attach = attach
       this.parameters = parameters
     }
   }
   let definitionFunctionWrapper = null
+  function addParameter({captureGroupRegexps, transformer, typeName}) {
+    const parameter = new Parameter(
+      typeName,
+      function() {},
+      captureGroupRegexps,
+      transformer
+    )
+    options.parameterRegistry.addParameter(parameter)
+  }
+  function addTransform({captureGroupRegexps, transformer, typeName}) {
+    // eslint-disable-next-line no-console
+    if (console !== 'undefined' && typeof console.error === 'function') {
+      // eslint-disable-next-line no-console
+      console.error('addTransform is obsolete and will be removed in a future version. Please use addParameter instead.')
+    }
+    addParameter({captureGroupRegexps, transformer, typeName})
+  }
```

Comment:
I'd prefer not to have function definitions in here. Can these be moved into helpers? Similar to how After/Before/defineStep work.